wire-ios-sync-engine/Tests/Source/Synchronization/Transcoders/ZMLastUpdateEventIDTranscoderTests.m

```diff
@@ -97,7 +107,12 @@ - (void)testThatItReturnsTheDownstreamSync
 - (void)testThatItCreatesTheRightDownstreamSync
 {
     // when
-    ZMLastUpdateEventIDTranscoder *sut = [[ZMLastUpdateEventIDTranscoder alloc] initWithManagedObjectContext:self.uiMOC applicationStatus:self.mockApplicationStatus syncStatus:self.mockSyncStatus];
+    LastEventIDRepository *lastEventIDRepository = [[LastEventIDRepository alloc] initWithUserID:self.userIdentifier
+                                                                                    userDefaults:NSUserDefaults.standardUserDefaults];
```

Comment:
same

Reply:
fixed in c8a59a522c5b2a06d5a9bf871a2a7e174175247d